repo/packages/K/kubernetes/4/config.json

```diff
@@ -194,6 +194,29 @@
           "description": "Which proxy mode to use: 'userspace' (older) or 'iptables' (faster). If the iptables proxy is selected, regardless of how, but the system's kernel or iptables versions are insufficient, this always falls back to the userspace proxy.",
           "type": "string",
           "pattern": "^(userspace|iptables)$"
+        },
+        "pki": {
```

Comment:
workaround in place by adding "disabled" as defaults for each of these key names. not iteal